src/components/Nav/Nav.js

```diff
@@ -1,8 +1,92 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+  faCartShopping,
+  faMagnifyingGlass,
+} from '@fortawesome/free-solid-svg-icons'
+
+import Modal from './Modal'
+import SearchModal from './SearchModal'
+import MENU_DATA from './navData'
+
+import './Nav.scss'
 
 const Nav = () => {
-  return <div>hello</div>
+  const [isHover, setIsHover] = useState(false)
+  const [isModal, setIsModal] = useState(false)
+  const [isSearchModal, setIsSearchModal] = useState(false)
+
+  const handleMouseOver = () => {
+    setIsHover(true)
+  }
+  const handleMouseOut = () => {
+    setIsHover(false)
+  }
+
+  return (
+    <>
+      <div className="nav">
+        <Link to="/">
+          <div className="logo">WePet</div>
+        </Link>
+        <div className="category">
+          {MENU_DATA.map(data => {
+            return (
+              <Link to={MENU_DATA.link}>
+                <span className="categoryName">{data.name}</span>
+              </Link>
+            )
+          })}
+        </div>
+        <div className="navIcon">
+          <FontAwesomeIcon
+            icon={faMagnifyingGlass}
+            onClick={() => {
+              setIsSearchModal(!isSearchModal)
```

Comment:
```suggestion
              setIsSearchModal(prev => !prev)
```
함수형 업데이트를 적용해주시는게 더 좋습니다~!